api_app/views.py

```diff
@@ -94,7 +94,10 @@ def _analysis_request(
     )
     ser.is_valid(raise_exception=True)
 
-    response_dict = ser.data
```

Comment:
This API will be replaced by the "multiple" APIs, also for files. I think that you made this change to support the same response data for files too. But I won't do that cause we will migrate file analysis to "multiple" too.

So, at the end, this API won't be used anymore by either the GUI or the clients. But I won't remove or change this to allow retrocompatibility for external integrations, like the one we have with Tines.

This is the reason why we cannot change input/output and URL for this API.